src/devTools.js

```diff
@@ -247,11 +250,14 @@ export const ActionCreators = {
   setMonitorState(monitorState) {
     return { type: ActionTypes.SET_MONITOR_STATE, monitorState };
   },
-  recomputeStates(committedState, stagedActions) {
+  recomputeStates(committedState, stagedActions, timestamps, currentStateIndex=-1, skippedActions={}) {
```

Comment:
See refactoring at https://github.com/gaearon/redux-devtools/compare/master...ioss:fix-for-recompute-states-2